Kconfig

```
@@ -1473,6 +1473,16 @@ menu "LVGL configuration"
 			int "Font manager name max length"
 			depends on LV_USE_FONT_MANAGER
 			default 32
+
+		config LVGL_VERSION_MAJOR
+			int "Major version"
+			default 9
+		config LVGL_VERSION_MINOR
+			int "Minor version"
+			default 2
```

What about to match a single line?
```suggestion
			default 2  #LVGL minor version
```